tests/robustness/model/history.go

```diff
@@ -364,38 +364,38 @@ func rangeResponse(kvs []*mvccpb.KeyValue, count int64, revision int64) EtcdNonD
 			},
 		}
 	}
-	return EtcdNonDeterministicResponse{EtcdResponse: EtcdResponse{Range: &result, Revision: revision}}
+	return MaybeEtcdResponse{EtcdResponse: EtcdResponse{Range: &result, Revision: revision}}
 }
 
-func failedResponse(err error) EtcdNonDeterministicResponse {
-	return EtcdNonDeterministicResponse{Err: err}
+func failedResponse(err error) MaybeEtcdResponse {
+	return MaybeEtcdResponse{Err: err}
 }
 
-func unknownResponse(revision int64) EtcdNonDeterministicResponse {
-	return EtcdNonDeterministicResponse{ResultUnknown: true, EtcdResponse: EtcdResponse{Revision: revision}}
+func unknownResponse(revision int64) MaybeEtcdResponse {
```

Comment:
How about renaming this function to `partialResponse()` in the follow ups?

Reply:
Done.